pkg/build/api/types.go

```diff
@@ -55,6 +55,9 @@ type BuildParameters struct {
 
 	// Output describes the Docker image the Strategy should produce.
 	Output BuildOutput `json:"output,omitempty"`
+
+	// Compute resource requirements to execute the build
+	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
Not setting any requirements or limits also communicates the same thing - empty slice already conveys that so nil struct doesn't improve our ability to see user intent.

> On Apr 21, 2015, at 10:50 PM, Ben Parees <notifications@github.com> wrote:
> 
> In pkg/build/api/types.go:
> 
> > @@ -55,6 +55,9 @@ type BuildParameters struct {
> >  
> >  	// Output describes the Docker image the Strategy should produce.
> >  	Output BuildOutput `json:"output,omitempty"`
> > +
> > +	// Compute resource requirements to execute the build
> > +	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
> derek and i discussed and i acquiesced, but i don't see how you can say that nil doesn't determine user intent...nil clearly indicates they did not set a quota. having an object w/ no values set is the one that would be confusing ("did they not set this, or did they half set it not understanding how to properly set it?")
> 
> —
> Reply to this email directly or view it on GitHub.
>